CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/AndroidContext.java

```diff
@@ -0,0 +1,59 @@
+package com.couchbase.CouchbaseLiteServ;
+
+import com.couchbase.javacommon.Context;
+import com.couchbase.javacommon.log.Log;
```

Comment:
What would you think of making the package: "com.couchbase.mobiletestkit...".  SOP is to narrow the namespace to the particular application so as not to pollute the base namespace.

Reply:
the suggested package name makes more sense. updated in code